scripts/build.py

```diff
@@ -1109,6 +1118,42 @@ def _verify_build(
             THIRD_PARTY_GENERATED_OUT_DIR, third_party_css_final_filename)])
 
 
+def _verify_file_count(
+        input_dirnames, output_dirnames):
+    """Verify number of files between source directory and final directory
+        matches after build process finishes.
+
+        Args:
+            input_dirnames: list(str). List of directory paths that contain
+                source files.
+            output_dirnames: list(str). List of directory paths that contain
+                built files.
+    """
+    for i in xrange(len(input_dirnames)):
+        # Make sure that all files in source directory and staging directory are
+        # accounted for.
+        _compare_file_count(input_dirnames[i], output_dirnames[i])
+
+
+def _verify_file_count_for_multiple_dirs(
+        input_dirnames_list, output_dirnames_list):
+    """Verify number of files between all directories in source directory
+    list and all directories in final directory list matches after build
+    process finishes.
+
+        Args:
+            input_dirnames_list: list(list(str)). List of list of directory
```

Comment:
why not just list(str)? Isn't it a list of dirnames?

Ditto below.

Reply:
Updated.